java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/registration/ConfluentCloudTopics.java

```diff
@@ -0,0 +1,119 @@
+package org.radarcns.schema.registration;
+
+import static org.radarbase.util.Strings.isNullOrEmpty;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import javax.validation.constraints.NotNull;
+
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.radarcns.schema.CommandLineApp;
+import org.radarcns.schema.util.SubCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConfluentCloudTopics extends AbstractTopicRegistrar {
+    private static final Logger logger = LoggerFactory.getLogger(ConfluentCloudTopics.class);
+
+    private final AdminClient kafkaClient;
+
+    /**
+     * Create Kafka topics registration object with given Confluent Cloud bootstrap server
+     * configurations.
+     *
+     * @param configPath File path of the Confluent-cloud config .
+     */
+    public ConfluentCloudTopics(@NotNull String configPath) throws IOException {
+
+        logger.info("Creating Kafka client with bootstrap servers {}", configPath);
+
+        this.kafkaClient = AdminClient.create(loadConfig(configPath));
+    }
+
+    @Override
+    AdminClient getKafkaClient() {
+        ensureInitialized();
+        return kafkaClient;
+    }
+
+    @Override
+    public void ensureInitialized() {
+        if (this.kafkaClient != null) {
+            throw new IllegalStateException("Kafka client is not initialized yet");
+        }
+    }
+
+    private Properties loadConfig(final String configFile) throws IOException {
+        Path filePath = Paths.get(configFile);
+        if (!Files.exists(filePath)) {
```

Comment:
You can skip this check, `Files.newInputStream` will throw `FileNotFoundException`.